.github/workflows/conventional-commits.yml

```
@@ -0,0 +1,188 @@
+      #       We require pull request titles to follow the Conventional Commits specification as outlined in our [documentation](https://github.com/SpecterOps/BloodHound/blob/main/rfc/bh-rfc-2.md).
+      #       Please review the required format and update your pull request title accordingly.
+      #       Thank you!
+
+      #       Details:
+
+      #       ```
+      #       ${{ steps.lint_pr_title.outputs.error_message }}
+      #       ```
+
+      # Delete a previous comment when the issue has been resolved
+      - if: ${{ steps.lint_pr_title.outputs.error_message == null }}
```

actionlint (1.7.8)

property "lint_pr_title" is not defined in object type {}

(expression)